src/autocomplete/databases/clickhouse/grammar/ClickHouseLexer.g4

```diff
@@ -8,342 +8,394 @@ lexer grammar ClickHouseLexer;
+ROWBINARYWITHDEFAULTS                      : R O W B I N A R Y W I T H D E F A U L T S;
+NATIVE                                     : N A T I V E;
+CAPNPROTO                                  : C A P N P R O T O;
+LINEASSTRING                               : L I N E A S S T R I N G;
+RAWBLOB                                    : R A W B L O B;
+MSGPACK                                    : M S G P A C K;
+MYSQLDUMP                                  : M Y S Q L D U M P;
+DWARF                                      : D W A R F;
+FORM                                       : F O R M;
+KEYED                                      : K E Y E D;
+RANDOMIZED                                 : R A N D O M I Z E D;
+USER_NAME                                  : U S E R UNDERSCORE N A M E;
+IP_ADDRESS                                 : I P UNDERSCORE A D D R E S S;
+FORWARDED_IP_ADDRESS                       : F O R W A R D E D UNDERSCORE I P UNDERSCORE A D D R E S S;
+CLIENT_KEY                                 : C L I E N T UNDERSCORE K E Y;
+CLIENT_KEY_OR_USER_NAME                    : C L I E N T UNDERSCORE K E Y UNDERSCORE O R UNDERSCORE U S E R N A M E;
+CLIENT_KEY_OR_IP_ADDRESS                   : C L I E N T UNDERSCORE K E Y UNDERSCORE O R UNDERSCORE I P UNDERSCORE A D D R E S S;
+QUERIES                                    : Q U E R I E S;
+QUERY_SELECTS                              : 'query_selects';
+QUERY_INSERTS                              : 'query_inserts';
+ERRORS                                     : E R R O R S;
+RESULT_ROWS                                : 'result_rows';
+RESULT_BYTES                               : 'result_bytes';
+READ_ROWS                                  : 'read_rows';
+READ_BYTES                                 : 'read_bytes';
+EXECUTION_TIME                             : 'execution_time';
+FAILED_SEQUENTIAL_AUTHENTICATIONS          : 'failed_sequential_authentications';
+LIMITS                                     : L I M I T S;
+TRACKING                                   : T R A C K I N G;
+ONLY                                       : O N L Y;
+CONST                                      : C O N S T;
+CHANGEABLE_IN_READONLY                     : C H A N G E A B L E UNDERSCORE I N UNDERSCORE R E A D O N L Y;
+OVERRIDABLE                                : O V E R R I D A B L E;
 
 JSON_FALSE : 'false';
 JSON_TRUE  : 'true';
 
 // Tokens
 
-IDENTIFIER          : (LETTER | UNDERSCORE) ( LETTER | UNDERSCORE | DEC_DIGIT)* | BACKQUOTE ( ~([\\`]) | (BACKSLASH .) | (BACKQUOTE BACKQUOTE))* BACKQUOTE | QUOTE_DOUBLE ( ~([\\"]) | (BACKSLASH .) | (QUOTE_DOUBLE QUOTE_DOUBLE))* QUOTE_DOUBLE;
+IDENTIFIER          : (LETTER | UNDERSCORE | DEC_DIGIT+ (LETTER | UNDERSCORE)) ( LETTER | UNDERSCORE | DEC_DIGIT)* | BACKQUOTE ( ~([\\`]) | (BACKSLASH .) | (BACKQUOTE BACKQUOTE))* BACKQUOTE | QUOTE_DOUBLE ( ~([\\"]) | (BACKSLASH .) | (QUOTE_DOUBLE QUOTE_DOUBLE))* QUOTE_DOUBLE;
```

Comment:
Clickhouse actually give ability to create subjects which start with digits, but they should have letter or underscore after digits